lib/pages/search_page.dart

```diff
@@ -3,6 +3,7 @@ import 'package:flutter/rendering.dart';
 import 'package:flutter_typeahead/flutter_typeahead.dart';
 import 'package:geolocator/geolocator.dart';
 import 'package:hap_map/api/location_api.dart';
+import 'package:hap_map/api/shake_api.dart';
```

Comment:
Is there a specific reason that the shake api is being used in the search page?

Reply:
The shake api is being initialized when the search page starts, so it can start detecting when the user shakes their device anytime after starting up the app and the listeners will work on all pages.

Reply:
Ahhh ok. Sounds good! LGTM